transcendence-app/src/game/game.service.ts

```diff
@@ -266,11 +296,15 @@ export class GameService {
       playerTwoLeftAt,
     };
     if (hasGameResumedNow) {
+      this.updateGameInfo(gameRoomId, {
+        ...game,
+        pausedAt: game.pausedAt - RESUME_GAME_DELAY_MS * 2,
```

Comment:
We set the paused at in the past to prevent a game from finishing right when about to resume because we update the game info with a delay.